x/nft/codec.proto

```diff
@@ -0,0 +1,49 @@
+syntax = "proto3";
+
+package nft;
+
+message NonFungibleToken {
+    bytes id = 1;
+    bytes owner = 2;
+    repeated ApprovalSet approvalSet = 3;
+}
+
+message ApprovalSet {
```

Comment:
While I like this name it does not reflect the contents, as the content is an object with 2 fields.
I would still settle for ActionApprovals, even though I don't like that name as much it's much more clear

Reply:
q: IMHO it is a Set type but how about `ApprovedActions`?

Reply:
btw. you are right with the Set